src/invert/fft_fftw.cxx

```diff
@@ -68,6 +68,7 @@ void fft_init(Options* options) {
   fft_init(fft_measurement_flag);
 }
 
+#if BOUT_HAS_FFTW
```

Comment:
Should we put only the function body in the `#if` to have a stabile ABI?

Reply:
This function is only used internally, so it should really be in an anonymous namespace. Let me fix that now too.